bitnami/keycloak/templates/statefulset.yaml

```
@@ -59,9 +59,45 @@ spec:
       {{- if .Values.podSecurityContext.enabled }}
       securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 8 }}
       {{- end }}
-      {{- if .Values.initContainers }}
+      {{- if or .Values.initContainers .Values.auth.tls.enabled }}
       initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.auth.tls.enabled }}
+      {{- $fullname := include "keycloak.fullname" . }}
+        - name: init-certs
+          image: {{ include "keycloak.auth.tls.image" . }}
+          imagePullPolicy: {{ .Values.auth.tls.image.pullPolicy | quote }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          command:
+          - /bin/sh
+          - -c
+          - |-
+            ID="${MY_POD_NAME#"{{ $fullname }}-"}"
+            if [[ -f "/certs/keycloak.truststore.jks" ]] && [[ -f "/certs/keycloak-${ID}.keystore.jks" ]]; then
+                mkdir -p /opt/bitnami/keycloak/certs
+                cp "/certs/keycloak.truststore.jks" "/opt/bitnami/keycloak/certs/keycloak.truststore.jks"
+                cp "/certs/keycloak-${ID}.keystore.jks" "/opt/bitnami/keycloak/certs/keycloak.keystore.jks"
+            else
+                echo "Couldn't find the expected Java Key Stores (JKS) files! They are mandatory when encryption via TLS is enabled."
+                exit 1
+            fi
+          env:
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                apiVersion: v1
+                fieldPath: metadata.name
+          {{- if .Values.auth.tls.resources }}
+          resources: {{- toYaml .Values.auth.tls.resources | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+          - name: certificates
+            mountPath: /certs
+          - name: shared-certs
+            mountPath: /opt/bitnami/keycloak/certs
+      {{- end }}
```

Missing `end` form the previous if:

```suggestion
            mountPath: /opt/bitnami/keycloak/certs
        {{- end }}
      {{- end }}
```

---

Line `100` ends the if statement in line `65` (I double checked and also counted the if statements). One thing I noticed is that some of the bitnami charts have an extra indent under the `container` and `initContainer` sections. If you look closely enough, the entire section is 2 spaces to the right of what actually gets deployed at runtime. I thought about correcting this but left it alone in case that was done on purpose.